tests/FakeItEasy.Tests/FakeFacadeTests.cs

```diff
@@ -106,7 +106,9 @@ public void Initialize_should_be_guarded()
             // Act
 
             // Assert
-             Expression<System.Action> call = () => this.facade.InitializeFixture(new object());
+#pragma warning disable CS0618 // Type or member is obsolete
+            Expression<System.Action> call = () => this.facade.InitializeFixture(new object());
+#pragma warning restore CS0618 // Type or member is obsolete

```

Comment:
If `FakeFacade.InitializeFixture` isn't marked obsolete, this warning suppression isn't necessary.